receiver/otlpreceiver/otlp_test.go

```diff
@@ -574,6 +574,31 @@ func TestGRPCInvalidTLSCredentials(t *testing.T) {
 		`failed to load TLS config: for auth via TLS, either both certificate and key must be supplied, or neither`)
 }
 
+func TestGRPCMaxRecvSize(t *testing.T) {
+	addr := testutil.GetAvailableLocalAddress(t)
+	sink := new(consumertest.TracesSink)
+
+	factory := NewFactory()
+	cfg := factory.CreateDefaultConfig().(*Config)
+	cfg.GRPC.NetAddr.Endpoint = addr
+	cfg.GRPC.MaxRecvMsgSizeMiB = 100
```

Comment:
Can you add a test showing what the expected behaviour is before settings`MaxRecvMsgSizeMiB` with the same test data?

Reply:
Added the test PTAL @bogdandrutu